.github/workflows/ci.yml

```diff
@@ -131,39 +133,28 @@ jobs:
           fail_ci_if_error: false
           verbose: true
 
-# fuzzing: # fuzzit is now offline
-#   name: Fuzz Testing
-#   runs-on: ubuntu-latest
-#   if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' }}
-#   steps:
-#     - name: Checkout source code
-#       uses: actions/checkout@v2
-#     - name: Install Rust Nightly
-#       uses: actions-rs/toolchain@v1
-#       with:
-#         toolchain: nightly
-#         default: true
-#     - name: Install Cargo Fuzz
-#       uses: actions-rs/install@v0.1
-#       with:
-#         crate: cargo-fuzz
-#         version: latest
-#         use-tool-cache: true
-#     - name: Bootstraping Grammars - Building
-#       uses: actions-rs/cargo@v1
-#       with:
-#         command: build
-#         args: --package pest_bootstrap
-#     - name: Bootstraping Grammars - Executing
-#       uses: actions-rs/cargo@v1
-#       with:
-#         command: run
-#         args: --package pest_bootstrap
-#     - name: Fuzz Testing - Regressions
-#       env:
-#         FUZZIT_API_KEY: ${{ secrets.FuzzitApiKey }}
-#       run: ./fuzzit.sh local-regression
-#     - name: Fuzz Testing - Fuzzing
-#       env:
-#         FUZZIT_API_KEY: ${{ secrets.FuzzitApiKey }}
-#       run: ./fuzzit.sh fuzzing
+            
+  publish:
+    # Only do this job if publishing a release
+    needs: build
```

Comment:
`build` isn't a job in this CI workflow. Also, the other tests may fail if they depend on `pest`. Either remove this line, or make sure all the tests pass and `needs: testing`?